database/index.js

```diff
@@ -11,17 +11,36 @@ let morePlaces = mongoose.Schema({
   rating: Number,
   numOfRatings: Number,
   description: String,
-  price: Number
+  price: Number,
+  similarPlaces: Array
 });
 
 let relatedPlaces = mongoose.model('MorePlaces', morePlaces);
 
 
+
 let getAllProperties = (callback) => {
-  relatedPlaces.find( (err, results) => {
+  relatedPlaces.find( {}, (err, results) => {
     if(err) {
       callback(err);
     } else {
+      let length = results.length;
+      for (let i = 0; i < length; i++) {
+        for (let j = 0; j < 12; j++) {
+          // console.log(results, 'results ')
+          // console.log(randomNum)
+          let recursive = () => {
```

Comment:
for function naming good practice, try to name it more descriptive so it can also provide some insights to what this function does.